tests/import_test.go

```diff
@@ -0,0 +1,35 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	testSuiteName           = "Importer Test Suite"
+	namespacePrefix         = "importer"
+	waitForCDIImportTimeout = 30 * time.Second
+)
+
+var _ = Describe(testSuiteName, func() {
+	f, err := framework.NewFramework(namespacePrefix)
+	if err != nil {
+		Fail("Unable to create framework struct")
+	}
+
+	Describe("Creating standard PVC should not trigger importer", func() {
```

Comment:
The title string here is pretty specific, which makes it seem like the existing `It()` is the only one that will ever be here.  If that's the case, it would be fine to extract the `It()` block to this scope and remove the nested `Describe()`.  And then just make the `It()` title a little more descriptive. (e.g. `It("should not perform CDI operations on PVCs without expected annotations", ... )`)

Reply:
Maybe I am a little confused at what goes in which level. The goal is the Describe is for all the import tests, and the It is just one of the tests, in particular the negative nothing changes test. Is this better?
```go
	Describe("PVC import tests", func() {
		It("Creating standard PVC should not trigger importer, and the PVC should remain empty", func() {
			pvc, err := f.CreatePVCFromDefinition(utils.NewPVCDefinition("no-import", "1G", nil, nil))
			Expect(err).ToNot(HaveOccurred())
			// Wait a while to see if CDI puts anything in the PVC.
			time.Sleep(waitForCDIImportTimeout)
			Expect(framework.VerifyPVCIsEmpty(f, pvc)).To(BeTrue())
			// Not deleting PVC as it will be removed with the NS removal.
		})
	})
```